models/surveySubmissions.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    name: "surveySubmissions",
+    schema: {
```

Comment:
@deepakr123 - Mode changes

      surveyId: "ObjectId", - Add index
      name: String,
      createdBy: String, - - Add index
      status: String,- Add index
      evidencesStatus: Array,
      evidences: Object,
      criteria: Array,
      themes: Array, - Not needed
      answers: Object,
      generalQuestions: Object,- Not needed
      completedDate: Date,
      solutionId: "ObjectId",- Add index
      solutionExternalId: String,- Add index
      programId: "ObjectId",- Add index
      programExternalId: String,- Add index
      submissionNumber: Number,- Not needed
      isRubricDriven: { type : Boolean, default: false },- Not needed
      startDate: Date,
      endDate: Date

Reply:
resolved